ab3d2_source/hires.s

```diff
@@ -7222,10 +7222,13 @@ tstwat:
 
 				tst.b	draw_UseWater_b
 				bne		texturedwater
-; tst.b draw_UseGouraudFlats_b						; FIXME: this effectively disables bumpmapped floors...
-										; opportunity to reenable and see what happens
-				bra		gouraudfloor
+; tst.b draw_UseGouraudFlats_b					; FIXME: this effectively disables bumpmapped floors...
+												; opportunity to reenable and see what happens
+				;bra		gouraudfloor
 
+				tst.b	Sys_CPU_68060_b
```

Comment:
I think this is called per line of the floor, so this may be less efficient than desired.